packages/blade-mcp-server/src/index.ts

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env node
+
+import { McpServer } from '@modelcontextprotocol/sdk/server/mcp.js';
+import { StdioServerTransport } from '@modelcontextprotocol/sdk/server/stdio.js';
+import { z } from 'zod';
+import { readFileSync, readdirSync } from 'fs';
+import { fileURLToPath } from 'url';
+import { dirname, resolve } from 'path';
+
+// Get package.json path and read version
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = dirname(__filename);
+const packageJson = JSON.parse(readFileSync(resolve(__dirname, '../package.json'), 'utf8'));
+
+// Dynamically read component files from knowledgebase directory
+const knowledgebasePath = resolve(__dirname, '../knowledgebase');
```

Comment:
We will be fetching from github right? because this I am assuming will break when we release it as NPM package because then it won't have rest of the monorepo files that you can read from